### PR TITLE
Change skill define names to be accurate

### PR DESCRIPTION
**Pull request by Shadowtail117**
## About the Pull Request

Changes the defines for:
- `SKILL_NONE` -> `SKILL_UNTRAINED`
- `SKILL_ADEPT` -> `SKILL_TRAINED`
- `SKILL_EXPERT` -> `SKILL_EXPERIENCED`
- `SKILL_PROF` -> `SKILL_MASTER`

This has no impact on front-end.

## Why It's Good For The Game

Easier to know what skill means what.

## Did you test it?

Compiles just fine.

## Changelog

N/A

### CODE REVIEW

**Comment by Ilysen** [comment]
Guessing this was just a simple find/replace. Fine by me, but deferring to @Cupax3 for final choice.

**Comment by Shadowtail117** [comment]
Correct guess. (After all, can't change the defines without changing where they're used.)

**Review by Cupax3 — DISMISSED** [review]
Scream test via testmerge authorized

File: code/__defines/skills.dm [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+#define SKILL_UNTRAINED     1
 #define SKILL_BASIC    2
-#define SKILL_ADEPT    3
-#define SKILL_EXPERT   4
-#define SKILL_PROF     5
-#define HAS_PERK       SKILL_NONE + 1
+#define SKILL_TRAINED    3
+#define SKILL_EXPERIENCED   4
+#define SKILL_MASTER     5
+#define HAS_PERK       SKILL_UNTRAINED + 1
```
Comment (Ilysen) [inline code/__defines/skills.dm]: This has been in for a hot minute, but I haven't noticed any problems, nor heard anything reported. I think this is good to go. Could you make the indentation on these uniform (similar to the old defines) for consistency? After that, I think I'm good signing off on this.

**Review by Ilysen — CHANGES_REQUESTED** [review]

**Comment by Cupax3** [comment]
I've not heard screaming, so it probably works fine. @Shadowtail117 please address the open conversation for merging.

**Comment by Shadowtail117** [comment]
Donezo

**Review by Ilysen — APPROVED** [review]

**Pull request merged**